--- accepted-reference9e011

+++ failed-comparisonb1d335

@@ -1,7 +1,7 @@

 first-pickup=actual-native-pickup
 tick=196
 map=0
-ids=nextAbilityID:0,nextAncientCryptosleepCasketGroupID:1,nextAnimalPenID:0,nextArchivedDialogID:0,nextAreaID:5,nextBattleID:0,nextBillID:0,nextCaravanID:0,nextCocoonGroupID:0,nextFactionID:11,nextGameConditionID:0,nextGeneID:68,nextHediffID:198,nextIdeoID:9,nextJobID:32,nextLetterID:0,nextLogID:0,nextLordID:1,nextMapID:1,nextMessageID:0,nextPassingShipID:0,nextPitGateIncidentID:0,nextPlanID:0,nextPreceptID:99,nextPresenceDemandID:0,nextPsychicRitualID:0,nextPsychicRitualToilID:0,nextQuestID:0,nextRitualObligationID:0,nextRitualRoleID:0,nextRoomID:0,nextShipJobID:0,nextSignalTagID:1,nextStorageGroupID:0,nextStructureSketchID:0,nextTaleID:1,nextThingID:40725,nextTransportShipID:0,nextTransporterGroupID:0,nextWorldFeatureID:62,nextWorldObjectID:93,nextZoneID:1
+ids=nextAbilityID:0,nextAncientCryptosleepCasketGroupID:1,nextAnimalPenID:0,nextArchivedDialogID:0,nextAreaID:5,nextBattleID:0,nextBillID:0,nextCaravanID:0,nextCocoonGroupID:0,nextFactionID:11,nextGameConditionID:0,nextGeneID:68,nextHediffID:198,nextIdeoID:9,nextJobID:32,nextLetterID:0,nextLogID:1,nextLordID:1,nextMapID:1,nextMessageID:0,nextPassingShipID:0,nextPitGateIncidentID:0,nextPlanID:0,nextPreceptID:99,nextPresenceDemandID:0,nextPsychicRitualID:0,nextPsychicRitualToilID:0,nextQuestID:0,nextRitualObligationID:0,nextRitualRoleID:0,nextRoomID:0,nextShipJobID:0,nextSignalTagID:1,nextStorageGroupID:0,nextStructureSketchID:0,nextTaleID:1,nextThingID:40726,nextTransportShipID:0,nextTransporterGroupID:0,nextWorldFeatureID:62,nextWorldObjectID:93,nextZoneID:1
 actors=Human37528@(98, 0, 117):1[25:HaulersDream_BulkHaul:A=Cloth37538:B=Cloth37539:C=invalid:qa=null:qb=Cloth37538,Cloth37539,Cloth37540:counts=17,25,41:count=1:forced=True:expiry=-1:encumbrance=False:takeDelay=0:giver=HaulersDream_HaulNearby
 driver=HaulersDream.JobDriver_BulkHaul
 cursor=1